### ISSUE TITLE
useVirtualFileSystem breaks typechecking

### ISSUE THREAD

**Issue by cancerberoSgx** [dsherret/ts-simple-ast#481]
Creating a Project with useVirtualFileSystem: true the getType() seems to be broken or atleast working differently than not using that configuration. See the example : 

[repro code in my playground](https://typescript-api-playground.glitch.me/#__editor_code__=%0Aimport%20Project%2C%20%7B%20ClassDeclaration%2C%20FunctionDeclaration%2C%20InterfaceDeclaration%2C%20MethodDeclaration%2C%20MethodSignature%2C%20Node%2C%20ParameterDeclaration%2C%20PropertyDeclaration%2C%20PropertySignature%2C%20TypeGuards%2C%20VariableDeclaration%20%7D%20from%20'ts-simple-ast'%0Aimport%20*%20as%20ts%20from%20'typescript'%0A%0Aexport%20function%20main(code%3A%20string%2C%20log%3A%20(msg%3A%20string)%20%3D%3E%20void)%20%7B%0A%20%20const%20project1%20%3D%20new%20Project(%7B%0A%20%20%20%20useVirtualFileSystem%3A%20true%0A%20%20%7D)%0A%20%20const%20sourceFile%20%3D%20project1.createSourceFile('src%2Findex.ts'%2C%20%60hello(%5B%5B1%2C2%5D%5D)%60)%0A%20%20const%20name1%20%3D%20sourceFile.getDescendants().find(TypeGuards.isCallExpression).getArguments().map(p%20%3D%3E%20p.getType().getText()).join('%2C%20')%0A%20%20log('name1%3A%20'%20%2B%20name1)%0A%0A%20%20const%20project2%20%3D%20new%20Project(%7B%0A%20%20%20%20%2F%2F%20useVirtualFileSystem%3A%20true%0A%20%20%7D)%0A%20%20const%20sourceFile2%20%3D%20project2.createSourceFile('src%2Findex2.ts'%2C%20%60hello(%5B%5B1%2C2%5D%5D)%60)%0A%20%20const%20name2%20%3D%20sourceFile2.getDescendants().find(TypeGuards.isCallExpression).getArguments().map(p%20%3D%3E%20p.getType().getText()).join('%2C%20')%0A%20%20log('name2%3A%20'%20%2B%20name2)%0A%0A%7D%0A__editor_input__=see%20the%20logs)

just in case I'm pasting the code: 

```ts
import Project, { TypeGuards } from 'ts-simple-ast';

const project1 = new Project({
  useVirtualFileSystem: true
})
const sourceFile = project1.createSourceFile('src/index.ts', `hello([[1,2]])`)
const name1 = sourceFile.getDescendants().find(TypeGuards.isCallExpression).getArguments().map(p => p.getType().getText()).join(', ')
console.log('name1: '+name1 )

const project2 = new Project({})
const sourceFile2 = project2.createSourceFile('src/index2.ts', `hello([[1,2]])`)
const name2 = sourceFile2.getDescendants().find(TypeGuards.isCallExpression).getArguments().map(p => p.getType().getText()).join(', ')
console.log('name2: '+name2 )

```

**Comment by dsherret** [comment]
Does it work when you include the lib.d.ts files in the virtual file system? https://dsherret.github.io/ts-simple-ast/setup/file-system#virtual-file-system

**Comment by cancerberoSgx** [comment]
Yes I've also noticed that when reading the compiler diagnostics - it's clear now, thanks and sorry for the trouble and missing that docs.

**Comment by dsherret** [comment]
@cancerberoSgx it's no trouble! It's a bonus to have this information in the issue tracker too because then people can find the solution easier when they run into the problem.
